resources/db-setup/init.sql

```diff
@@ -0,0 +1,3229 @@
+-- workshop_setup.sql
```

Comment:
Ich bin nicht ganz sicher bezüglich den Lizenzen aber eventuell wollen wir das hier noch hinzufügen?


```suggestion
/*
This database setup code is based on the Microsoft SQL Server Sample Code.
Copyright (c) Microsoft Corporation. All rights reserved.

Licensed under the MIT License.
Full license text available in the ATTRIBUTION.md file in the repository root.
*/
-- workshop_setup.sql
```
Ich mache noch einen zweiten Kommentar mit den Attributions :)